### PR TITLE
add toleration to all taints + affinity rules for csi-node

### PR DESCRIPTION
**Pull request by i-prudnikov**

### CODE REVIEW

File: deploy/kubernetes/setup-upcloud-csi.yaml [hunk 1]
```diff
@@ -220,6 +220,16 @@ spec:
       priorityClassName: system-node-critical
       serviceAccount: csi-upcloud-node-sa
       hostNetwork: true
+      tolerations:
+        - effect: NoSchedule
+          operator: Exists
```
Comment (peknur) [inline deploy/kubernetes/setup-upcloud-csi.yaml:225]: Should we use double quotes with these values?

**Review by peknur — APPROVED** [review]
Left a non blocking comment, but other than that looks good :+1:

File: deploy/kubernetes/setup-upcloud-csi.yaml (hunk 1 above)
Comment (peknur) [reply deploy/kubernetes/setup-upcloud-csi.yaml:225]: Never mind my previous comment. It seems that our spec is not that consistent with these as I first thought.

Comment (i-prudnikov) [reply deploy/kubernetes/setup-upcloud-csi.yaml:225]: Well, I also not 100% sure what is the standard, but in majority of the cases we are not using quotes

**Pull request merged**